src/lib/deno-client.ts

```
@@ -0,0 +1,32 @@
+  }
+
+  static async healthCheck() {
+    const response = await fetch(`${this.baseUrl}/health`, {
+      headers: {
+        'Authorization': `Bearer ${this.apiKey}`,
+      },
+    });
+
+    return response.ok;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)